### PR TITLE
#326 Add cursor pointer in clickable elements

### PR DESCRIPTION
**Pull request by khushi818**
## Issues #326 
pointer added to 
- Buttons 
- input with type submit
- list elements for cards in user management

Instead of changing CSS in each style.css file, I created a global.css **as per the suggestions**

Note : Imports are replaced with links in html

### CODE REVIEW

File: featureFlag/style.css [hunk 1]
```diff
@@ -19,6 +19,10 @@ body {
   align-items: center;
 }
 
+button {
```
Comment (sahsisunny) [inline featureFlag/style.css:22]: Can we use `class` name instead of `tag`?

Comment (khushi818) [reply featureFlag/style.css:22]: Yes, we can but buttons are defined at the top level and could be overridden very easily. If new buttons or input with submit type is added, they will automatically have a pointer cursor and could be changed by defining using their class or id.

**Review by TanishqSingla — APPROVED** [review]

File: featureFlag/style.css (hunk 1 above)
Comment (sahsisunny) [reply featureFlag/style.css:22]: If you are setting a universal properties for all buttons and elements then you can create a global CSS file also

Comment (khushi818) [reply featureFlag/style.css:22]: got it thanks 👍

Comment (khushi818) [reply featureFlag/style.css]: @sahsisunny I did the changes you asked

**Review by sahsisunny — DISMISSED** [review]

File: featureFlag/style.css [hunk 2]
```diff
@@ -1,3 +1,5 @@
+@import url('../global.css');
```
Comment (TanishqSingla) [inline featureFlag/style.css:1]: Import statement might not be ideal here, can this be changed?
refer [this](https://www.debugbear.com/blog/avoid-css-import#:~:text=The%20CSS%20%40import%20rule%20can,to%20take%20longer%20to%20render.)

File: goal/style.css [hunk 3]
```diff
@@ -1,3 +1,5 @@
+@import url('../global.css');
```
Comment (TanishqSingla) [inline goal/style.css:1]: same as above

File: online-members/online-members.css [hunk 4]
```diff
@@ -1,3 +1,5 @@
+@import url('../global.css');
```
Comment (TanishqSingla) [inline online-members/online-members.css:1]: same as above

File: profile/style.css [hunk 5]
```diff
@@ -1,3 +1,5 @@
+@import url('../global.css');
```
Comment (TanishqSingla) [inline profile/style.css:1]: same as above

File: style.css [hunk 6]
```diff
@@ -1,8 +1,11 @@
+@import url('../global.css');
```
Comment (TanishqSingla) [inline style.css:1]: same as above

File: task/style.css [hunk 7]
```diff
@@ -1,3 +1,5 @@
+@import url('../global.css');
```
Comment (TanishqSingla) [inline task/style.css:1]: same as above

File: taskEvents/style.css [hunk 8]
```diff
@@ -1,3 +1,5 @@
+@import url('../global.css');
```
Comment (TanishqSingla) [inline taskEvents/style.css:1]: same as above

File: users/details/style.css [hunk 9]
```diff
@@ -1,3 +1,5 @@
+@import url('../../global.css');
```
Comment (TanishqSingla) [inline users/details/style.css:1]: same as above

File: users/style.css [hunk 10]
```diff
@@ -1,3 +1,5 @@
+@import url('../global.css');
```
Comment (TanishqSingla) [inline users/style.css:1]: same as above

File: wallet/style.css [hunk 11]
```diff
@@ -1,3 +1,5 @@
+@import url('../global.css');
```
Comment (TanishqSingla) [inline wallet/style.css:1]: same as above

**Review by TanishqSingla — CHANGES_REQUESTED** [review]

**Review by TanishqSingla — APPROVED** [review]

**Review by sahsisunny — APPROVED** [review]
Looks good to me

**Review by kotesh-arya — APPROVED** [review]
Looks Good To Me 👍🏻

File: featureFlag/style.css (hunk 2 above)
Comment (kotesh-arya) [reply featureFlag/style.css]: > Import statement might not be ideal here, can this be changed? refer [this](https://www.debugbear.com/blog/avoid-css-import#:~:text=The%20CSS%20%40import%20rule%20can,to%20take%20longer%20to%20render.)

Great observation and well-suggested guideline @TanishqSingla  🙌🏻

**Pull request merged**